docker/oap-es7/docker-entrypoint.sh

```diff
@@ -18,478 +18,18 @@
-    # Unit seconds, sync period. Default fetch every 60 seconds.
-    period : \${SW_CONFIGURATION_NACOS_PERIOD:5}
-    # the name of current cluster, set the name if you want to upstream system known.
-    clusterName: \${SW_CONFIGURATION_NACOS_CLUSTER_NAME:default}
-EOT
-}
-
-generateConfigurationZookeeper() {
-    cat <<EOT >> ${var_application_file}
-configuration:
-  zookeeper:
-    period: \${SW_CONFIGURATION_ZOOKEEPER_PERIOD:60} # Unit seconds, sync period. Default fetch every 60 seconds.
-    nameSpace: \${SW_CONFIGURATION_ZOOKEEPER_NAMESPACE:/default}
-    hostPort: \${SW_CONFIGURATION_ZOOKEEPER_HOST_PATH:localhost:2181}
-    #Retry Policy
-    baseSleepTimeMs: \${SW_CONFIGURATION_ZOOKEEPER_BASE_SLEEP_TIME_MS:1000} # initial amount of time to wait between retries
-    maxRetries: \${SW_CONFIGURATION_ZOOKEEPER_MAX_RETRIES:3}3 # max number of times to retry
-EOT
-}
-
-generateConfigurationGRPC() {
-    cat <<EOT >> ${var_application_file}
-configuration:
-  grpc:
-    host: \${SW_CONFIGURATION_GRPC_HOST:127.0.0.1}
-    port: \${SW_CONFIGURATION_GRPC_PORT:9555}
-    period: \${SW_CONFIGURATION_GRPC_PERIOD:60}
-    clusterName: \${SW_CONFIGURATION_GRPC_CLUSTER_NAME:"default"}
-EOT
-}
-
-generateConfigurationConsul() {
-    cat <<EOT >> ${var_application_file}
-configuration:
-  consul:
-    # Consul host and ports, separated by comma, e.g. 1.2.3.4:8500,2.3.4.5:8500
-    hostAndPorts: \${SW_CONFIGURATION_CONSUL_ADDRESS:127.0.0.1:8500}
-    # Sync period in seconds. Defaults to 60 seconds.
-    period: \${SW_CONFIGURATION_CONSUL_PERIOD:60}
-EOT
-}
-
-generateTelemetryNone() {
-    cat <<EOT >> ${var_application_file}
-telemetry:
-  none:
-EOT
-}
-
-generateTelemetryPrometheus() {
-    cat <<EOT >> ${var_application_file}
-telemetry:
-  prometheus:
-    host: \${SW_TELEMETRY_PROMETHEUS_HOST:0.0.0.0}
-    port: \${SW_TELEMETRY_PROMETHEUS_PORT:1234}
-EOT
-}
-
-generateTelemetrySo11y() {
-    cat <<EOT >> ${var_application_file}
-telemetry:
-  so11y:
-    prometheusExporterEnabled: \${SW_TELEMETRY_SO11Y_PROMETHEUS_ENABLED:true}
-    prometheusExporterHost: \${SW_TELEMETRY_PROMETHEUS_HOST:0.0.0.0}
-    prometheusExporterPort: \${SW_TELEMETRY_PROMETHEUS_PORT:1234}
-EOT
-}
-
-validateVariables() {
-    name=$1; value=$2; list=$3
-    valid=false
-    for c in ${list} ; do
-        if [[ "$c" = "$value" ]]; then
-            valid=true
-        fi
-    done
 
-    if ! ${valid}; then
-        echo "Error: $name=$value please specify $name = $list"
-        exit 1
-    fi
-}
-
-generateApplicationYaml() {
-    # validate
-    [[ -z "$SW_CLUSTER" ]] && [[ -z "$SW_STORAGE" ]] && [[ -z "$SW_CONFIGURATION" ]] \
-        && [[ -z "$SW_TELEMETRY" ]] \
-        && { echo "Error: please specify \"SW_CLUSTER\" \"SW_STORAGE\" \"SW_CONFIGURATION\" \"SW_TELEMETRY\""; exit 1; }
-
-    validateVariables "SW_CLUSTER" "$SW_CLUSTER" "standalone zookeeper kubernetes consul etcd nacos"
-
-    validateVariables "SW_STORAGE" "$SW_STORAGE" "elasticsearch h2 mysql influxdb"
```

Comment:
FYI @wu-sheng and @hanahmily , here is another bug in the Docker image, with this, the docker image for es7 never works